### DISCUSSION TITLE
General requests for the community

### DISCUSSION THREAD

**Discussion by ferraridamiano** [ferraridamiano/ConverterNOW#57] [General]
Hi guys, this projects exists because of you. ...and sometimes I need also your work to get this app better and better. I ask you to subscribe to this discussion if you want to help in the future.
It's really important, I can't do everything alone.

Thanks in advance 💚

**Comment by ferraridamiano** [comment]
I just finished to rewrite all the app. It took several days of work but it is finally here. As you might think this work could have introduced some bugs, and I need your help in order to find them out.
This version of the app doesn't introduce new feature or fix known bugs.
I attached 3 files (depending on your phone, usually arm64). I ask you to install the apk, try to use the app at 100% of what it can do (convert, reorder units, reorder property, open, close, change settings, and so on). If you noticed or not some bugs please let me know.
Thank you!

Maybe @u2code you are interested 🤞

[Click here to download the zip file](https://github.com/ferraridamiano/ConverterNOW/files/5973927/release.zip)

**Reply by u2code** [reply]
Work in progress! ;-) I'll send you a feedback tomorrow.

**Reply by u2code** [reply]
Hello @ferraridamiano 
I was using the version "armeabi-v7a". 

What **works as expected**:
- The app installs properly, runs stable and never crashed.
- German translation is matched correctly for my device.
- I can change any settings as expected.
- I can reorder all things as expected.
- Search works fine. But I can only search for example for "seconds" and not for "Time". Of course the search works fine with German translations.

What does **not work**:
- Symbols used for "Time" are not correct. Instead of symbols for time there are symbols from "Area"-section. Symbol for seconds is m², etc.
- Conversions for Force and Numerical Systems doesn't work at all. Both worked fine in the F-Droid version 2.4.3 for me. If I type anything in any of the fields, there is no action at all. I even uninstalled the app and tried it again without making any changes in the settings.
- I found some small misspellings in the German translation. I'll fix it soon.

I hope I could help you with my feedback.

**Reply by fux0r2009** [reply]
I tested It on **arm64-v8a** on Android 10 and I run into the same bugs reported by @u2code

**Reply by ferraridamiano** [reply]
Thanks for your work @u2code @fux0r2009 ! I will fix what you reported in the next <s>week</s> hours!

**Reply by ferraridamiano** [reply]
I fixed all the bugs you reported. This version should solve all the problems.
Thank you again for your help!

[Click here to download the zip file](https://github.com/ferraridamiano/ConverterNOW/files/5975717/release.zip)

**Reply by u2code** [reply]
Great work! You are pretty fast. I tested **armeabi-v7a** on Android 10 / LineagageOS 17.1 and everything seems to work fine so far.

**Reply by fux0r2009** [reply]
> I fixed all the bugs you reported. This version should solve all the problems.
> Thank you again for your help!
> 
> [Click here to download the zip file](https://github.com/ferraridamiano/ConverterNOW/files/5975717/release.zip)

 I tested arm64-v8a on Android 10 and all bugs are fixed, Wonderful jub @ferraridamiano

**Reply by ferraridamiano** [reply]
Thank you guys!

**Comment by ferraridamiano** [comment]
Hi guys! Recently I became aware of a standard denomination for currencies name thanks to @cosify (view this [PR](https://github.com/ferraridamiano/ConverterNOW/pull/69)). Now the english names for currencies should be perfect, but all others languages not. As cosify said [this repo](https://github.com/umpirsky/currency-list) could help us in the translations. Do you some of you wants to rename all the others currencies name in the standard names? If so, write it below so other people know that someone is already doing this.
Thanks in advance 💚

**Reply by u2code** [reply]
...work in progress for all translations which are already done. I'll be ready in few hours.

**Reply by ferraridamiano** [reply]
Thank you a lot @u2code !!!

**Comment by ferraridamiano** [comment]
Hi @u2code, and sorry if I always ask to you. A german guy wrote me an email and said that the translation of "quintal" in german is not "Zentner", because 1 Zentner = 50 kg and not 100 kg. And that also the translation of "tonne" is misspelled. Can you confirm this? If so I will fix it.
Thank you 😊

**Reply by u2code** [reply]
Hi @ferraridamiano

Well the answer is not really easy. Actually this guy is right, that in Germany, or at least the south part of it, **Zentner** means 50 kg. It stands for 100 x _Pfund_ (500 gram). But in other German speaking countries like Austria and I think also Switzerland **Zentner** means always 100 kg. Though 100 kg means **Doppelzentner** in Germany, but this word is not used in Austria and or Switzerland.

To make thinks clear we should maybe use the term **Zentner / Quintal (100 kg)**?

**Reply by ferraridamiano** [reply]
Thank you @u2code! Yeah we can use that translation!
